src/components/Autosuggest/Autosuggest.js

```diff
@@ -185,12 +185,8 @@ export class Autosuggest extends React.Component {
 
         switch (changes.type) {
```

Comment:
The modal is opened after switching focus browser/another application, steps for reproduce:
1) Open `LocationSelector`
2) Choose some location
3) Close `LocationSelector`
4) Switch to another application
5) Switch back on the browser tab. `LocationSelector` will be opened

Reply:
I created a separate issue for this story.
https://jira.textkernel.nl/browse/ONEUI-197